### PR TITLE
fix: Replaced FaviconWebpackPlugin to WebappWebpackPlugin

### LINKED ISSUE #11: Favicons plugin is not working with webpack 4
The `faviconPlugin` is not working with webpack 4. 

It would be better to remove this option for timebeing.

Followup issue: https://github.com/jantimon/favicons-webpack-plugin/issues/108

### PR DESCRIPTION
**Pull request by rishabh3112**
Resolves #11 
Requires a check @sendilkumarn

### CODE REVIEW

File: generator.js [hunk 1]
```diff
@@ -136,8 +136,8 @@ module.exports = class WebpackGenerator extends Generator {
 			.then(answer => {
 				if (answer) {
 					favPath = answer['favPath'];
-					this.options.env.configuration.dev.topScope.push('const FaviconsWebpackPlugin = require("favicons-webpack-plugin");');
-					this.dependencies.push("favicons-webpack-plugin");
+					this.options.env.configuration.dev.topScope.push("const WebappWebpackPlugin = require('webapp-webpack-plugin');");
```
Comment (sendilkumarn) [inline generator.js]: Can you please revert the `quotes` here ?

Comment (rishabh3112) [reply generator.js]: Sure, In next commit

**Comment by rishabh3112** [comment]
@sendilkumarn can you create a basic scaffold to check whether this fix actually solves the problem or not?
Just for a cross-check

**Comment by rishabh3112** [comment]
@sendilkumarn I have cross checked it and I think it's ready to merge.

**Comment by sendilkumarn** [comment]
I am on it 👍 I will merge this today 😊

**Comment by rishabh3112** [comment]
It will certainly cause a conflict with #24 @sendilkumarn. I have integrated #22 into #24.

**Comment by sendilkumarn** [comment]
Closing this in favor of #24

**Pull request closed without merging**